PWGJE/EMCALJetTasks/AliAnalysisTaskRhoSparse.h

```diff
@@ -66,9 +66,10 @@ class AliAnalysisTaskRhoSparse : public AliAnalysisTaskRhoBase {
   		);
 
   void             UserCreateOutputObjects();
-  void             SetExcludeLeadJets(UInt_t n)         { fNExclLeadJets = n       ; }
-  void             SetExcludeOverlapJets(Bool_t input)  { fExcludeOverlaps = input ; }
-  void             SetRhoCMS(Bool_t cms)                { fRhoCMS = cms            ; }
+  void             SetExcludeLeadJets(UInt_t n)           { fNExclLeadJets = n       ; }
+  void             SetExcludeOverlapJets(Bool_t input)    { fExcludeOverlaps = input ; }
+  void             SetRhoCMS(Bool_t cms)                  { fRhoCMS = cms            ; }
+  void             SetAreaCalculationDetails(Bool_t inputA, Bool_t inputB){ fUseTPCArea = inputA ; fExcludeAreaExcludedJets = inputB; }
```

Comment:
Please use self-explaining parameter names. inputA and inputB are not easy to digest.